dbcon/ddlpackage/ddl.l

```diff
@@ -65,7 +65,7 @@ double_quote \"
 grave_accent `
 
 comment ("--"{non_newline}*)
-extended_ident_cont [A-Za-z\200-\377_0-9\$#,()\[\].;\:\+\-\*\/\%\^\<\>\=!&|@\\]
+extended_ident_cont [ A-Za-z\200-\377_0-9\$#,()\[\].;\:\+\-\*\/\%\^\<\>\=!&|@\\]
```

Comment:
We better clean up lexer removing 'column_ident_quoted' and 'ident_w_spaces' patterns given this change. JFYI CS works as expected after I removed both patterns and checked regression 023 and MCOL-2219 case.